### ISSUE TITLE
const is writable from inner block

### ISSUE THREAD

**Issue by teppeis** [babel/babel#1494]
Input:
```javascript
const b = 1;
{
  const b = 1;
  {
    b = 2; // should be an error, but pass
  }
  console.log(b); // 2
}
```
Output:
```javascript
"use strict";

var b = 1;
{
  var _b = 1;
  {
    _b = 2; // should be an error, but pass
  }
  console.log(_b); // 2
}
```
http://babeljs.io/repl/#?experimental=false&evaluate=true&loose=false&spec=false&playground=true&code=const%20b%20%3D%201%3B%0A%7B%0A%20%20const%20b%20%3D%201%3B%0A%20%20%7B%0A%20%20%20%20b%20%3D%202%3B%20%2F%2F%20should%20be%20an%20error%2C%20but%20pass%0A%20%20%7D%0A%20%20console.log(b)%3B%20%2F%2F%202%0A%7D

**Comment by sebmck** [comment]
Looks like I fixed this in `master` when I rewrote the constants transformer for better performance. It'll be available in the next release. Thanks for the report!

**Comment by teppeis** [comment]
I'll try next release. Thanks!

**Comment by teppeis** [comment]
Fixed at v5.3.1. thanks
